study.md

```diff
@@ -134,15 +134,15 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a brief description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+Long ago, I memorized HTTP to stand for "Hyper Text Transfer Protocol". I understand it to be a standard from which different browser applications can use to interpret and display data.
 ```
 
 ## Describe what a client is and what a server is
 
  What is a client and what is a server? How do they interact with each other?
 
 ```md
-<!-- your answer here -->
+Both clients and servers are computers. I see the names as being the role that they play in relation to each other, The computer that sends requests to another computer is called the client. The computer that processes the request is called the server. After the server has processed the request, it sends a response back to the client. The client processes the response.
```

Comment:
This is good, the client can be the specific program like Chrome or Safari, not just the computer.